pure.zsh

```diff
@@ -292,11 +292,12 @@ prompt_pure_async_vcs_info() {
 prompt_pure_async_git_dirty() {
 	setopt localoptions noshwordsplit
 	local untracked_dirty=$1
+	local untracked_git_mode=$(command git config --get status.showUntrackedFiles) || 'normal'
 
 	if [[ $untracked_dirty = 0 ]]; then
 		command git diff --no-ext-diff --quiet --exit-code
 	else
-		test -z "$(command git status --porcelain --ignore-submodules -unormal)"
+		test -z "$(command git status --porcelain --ignore-submodules -u${untracked_git_mode})"
```

Comment:
Why don't you just remove this flag?

Reply:
`-unormal` is much faster than `-uall` because it doesn't recurse into untracked directories, so it's sensible to default to `-unormal` instead of git's default which is `-uall`. For the purposes of a dirty bit, recursing is entirely unnecessary.

Reply:
Good point - `-uall` should not be allowed.